docs/usage/configuration-options.md

```diff
@@ -1683,10 +1683,13 @@ Possible values and meanings:
 
 - `auto`: Renovate will autodetect the best setting. Defaults to `conflicted` unless the repository has a setting requiring PRs to be up to date with the base branch
 - `never`: Renovate will never rebase the branch
-- `conflicted`: Renovate will rebase only if the branch is conflicted
+- `conflicted`: Renovate will rebase only if the branch is conflicted.
```

Comment:
```suggestion
- `conflicted`: Renovate will rebase only if the branch is conflicted
```

We chose not to have dots at the end of a list item, so I'm removing it with this suggestion. 😉